app-crypt/moolticute/files/moolticuted.init

```diff
@@ -0,0 +1,18 @@
+#!/sbin/openrc-run
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+command=/usr/bin/moolticuted
+command_args=""
+pidfile=/var/run/moolticuted.pid
+
+depend() {
+	need local udev
```

Comment:
You aren't really supposed to `need local`, ever. Otherwise, your script can just fail randomly depending on the state of sysadmin's `local`.

If you want it to run as late as possible, use `after local`.

Reply:
Since I copied that from another ebuild and wasn't sure, I choose to remove `local`.